shed/event_streams.py

```diff
@@ -464,7 +464,16 @@ def issue_event(self, outputs):
                                        for (output_name, desc), output in
                                        zzip(self.output_info, outputs)})
             else:
-                new_event.update(data=outputs['data'])
+                if outputs is None:
+                    outputs = {}
+                elif not isinstance(outputs, dict):
+                    errormsg = "Error, outputs is not a dict. Can't continue"
```

Comment:
my problem is the indentation. i think it would keep the indentation if i did that. I tested and I have the same issue.
```python
if True:
    a = "fkldsjaflsdjf\
    flkdsjfasd\
     kfldajslfds"
     print(a)
``
gives 
```
fkldsjaflsdjf    flkdsjfasd    kfldajslfds
```
and 
```python
if True:
     b = '''klfjsdalfjdsafd
     kdfjaslkfjdsakl
     lkdjfalsjdsa'''
     print(b)
```
gives

```
klfjsdalfjdsafd
    kdfjaslkfjdsakl
    lkdjfalsjdsa
```
(both have that indentation)
I'm not sure if there's a way around that, haven't really checked

Also,